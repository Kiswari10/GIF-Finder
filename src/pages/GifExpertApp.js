import React, { useState } from "react";
import { AddCategory } from "../components/AddCategory";
import { GifGrid } from "../components/GifGrid";
import {Container} from "react-bootstrap";

export const GifExpertApp = () => {
  const [categories, setCategories] = useState(["Naruto"]);
  return (
    <Container>
      <h1 className="logo">GIF-Finder</h1>
      <AddCategory setCategories={setCategories} />
      {categories.map((category) => (
        <GifGrid key={category} category={category} />
      ))}
    </Container>
  );
};
