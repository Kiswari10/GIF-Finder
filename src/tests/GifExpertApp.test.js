import React from "react";
import { shallow } from "enzyme";

import { GifExpertApp } from "../pages/GifExpertApp";

describe("Test en <GifExpertApp />", () => {
  test("debe retornar el componente correctamente", () => {
    const wraper = shallow(<GifExpertApp />);
    expect(wraper).toMatchSnapshot();
  });
});
