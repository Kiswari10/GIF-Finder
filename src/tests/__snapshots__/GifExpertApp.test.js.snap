// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test en <GifExpertApp /> debe retornar el componente correctamente 1`] = `
<Container
  fluid={false}
>
  <h1
    className="logo"
  >
    GIF-Finder
  </h1>
  <AddCategory
    setCategories={[Function]}
  />
  <GifGrid
    category="Naruto"
    key="Naruto"
  />
</Container>
`;
